src/Providers/Trustpilot/Request/Merchant/InviteRequest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\Trustpilot\Request\Merchant;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use PodPoint\Reviews\Request\BaseRequest;
+
+/**
+ * Class InviteRequest
+ * @package PodPoint\Reviews\Providers\Trustpilot\Request\Service
+ */
+class InviteRequest extends BaseRequest
+{
+    /**
+     * Builds the request.
+     *
+     * @return Request
+     */
+    public function getRequest(): Request
+    {
+        $businessUnitId = $this->getOption('businessUnitId');
+
+        $this->options['serviceReviewInvitation'] = [
+            'preferredSendTime' => $this->options['preferredSendTime']
+        ];
+
+        unset($this->options['preferredSendTime']);
```

Comment:
Left a comment related to this above:

I think we should not pass this as bottom leve, we should  match the structure of the payload that we actually send out instead